pipelines/shared-template.yml

```diff
@@ -0,0 +1,99 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+parameters:
+    - name: templateType
+      type: string
+      default: UnOfficial
+      values:
+          - UnOfficial
+          - Official
+
+# The `resources` specify the location and version of the 1ES PT.
+resources:
+    repositories:
+        - repository: 1esPipelines
+          type: git
+          name: 1ESPipelineTemplates/1ESPipelineTemplates
+          ref: refs/tags/release
+extends:
+    # The pipeline extends the 1ES PT which will inject different SDL and compliance tasks.
+    # For non-production pipelines, use "Unofficial" as defined below.
+    # For productions pipelines, use "Official".
+    ${{ if eq(parameters.templateType, 'Official') }}:
+        template: v1/1ES.Official.PipelineTemplate.yml@1esPipelines
+    ${{ else }}:
+        template: v1/1ES.Unofficial.PipelineTemplate.yml@1esPipelines
+    parameters:
+        settings:
+            skipBuildTagsForGitHubPullRequests: true
+        # Update the pool with your team's 1ES hosted pool.
```

Comment:
Can this comment be reasonably removed since we've already made the update?
```suggestion
```

Reply:
Done